testing/internal/e2e/boundary/bucket.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package boundary
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/boundary/api/storagebuckets"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+// CreateNewStorageBucket uses boundary CLI to create new Storage Bucket in the us-east-1 region
```

Comment:
Could the region be a parameter, even if we end up setting an environment variable before making the call?